lib/cuckoo/core/database.py

```diff
@@ -957,7 +957,7 @@ def filter_machines_by_arch(machines, arch):
         Allow x64 machines to be returned when requesting x86.
         """
         if arch:
-            if arch == "x86":
+            if arch == ["x86"] or set(arch) == {"x64", "x86"}:
```

Comment:
can we just use `if "x86" in arch:`?

Reply:
brilliant, see https://github.com/kevoreilly/CAPEv2/pull/1085/commits/a74f383bd09ef41d9ae9a03839e6658813365967